### DISCUSSION TITLE
Create3 Crashed with Red Ring

### DISCUSSION THREAD

**Discussion by slowrunner** [iRobotEducation/create3_docs#502] [Questions]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Mac OS, Fusion VM, Ubuntu 20.04

### Which version of ROS 2 is installed on your computer?

Humble

### Which firmware version is installed on your robot?

H.2.3

### Which RMW is your robot running?

FastRTPS

### Is the robot connected to a network? If so, what is the network type?

Yes: 2.4GHz Home WiFi 

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

Yes

### What is the Adapter Board's USB/BLE Toggle currently switched to?

Bluetooth (default)

### Describe your question.

Create3 Crashed To Solid Red Ring

My Create3 had been running with no issues for two weeks or so, (since the reboots to install Humble 2.3 firmware).

Tonight, I issued a rotate_angle 3.14 rad (180 deg) action goal and it succeeded.  

Then I attempted to issue a dock action goal, but received "waiting for action server" way too long, so I looked in the log.  The rotate_angle success was listed, followed by one CPU 95% entry, but the log would not update and the site became unresponsive.

I checked the Create3 and found it showing the solid red ring.

Next, I held the power button for a long time, until the Create3 turned off.  Then pressed it again and the Create3 came alive spinning white light (like it was charging but it was not on the dock).  I held the power button again until it shut down.

Pressing the power button again, the Create3 booted up normally and appears fully recovered.

Attaching the log from after the reboot - prior log could not be captured:
[messages.txt](https://github.com/iRobotEducation/create3_docs/files/13848460/messages.txt)

**Comment by slowrunner** [comment]
Closing this as I am now on H.2.4 Firmware.  Create3 has not missed sending a Dock goal result succeeded (nor crashed) on the new firmware.

**Reply by shamlian** [reply]
Works for me, please re-open if you encounter a problem on H.2.4.
